src/System.Security.Permissions/src/System/SystemException.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System
```

Comment:
This type is just temporary here, I assume, and will actually end up living in System.Runtime? (I'm also assuming that's why it's missing ctors and the rest of its exposed surface area.)